Formula/bun.rb

```diff
@@ -0,0 +1,65 @@
+class Bun < Formula
+  desc "Fast all-in-one JavaScript runtime"
+  homepage "https://bun.sh"
+  # pull from git tag to get submodules
+  url "https://github.com/oven-sh/bun.git",
+    tag:      "bun-v0.1.2",
+    revision: "1ee94d5bd261b4cfad7798f0e119499f351b93f3"
+  license "MIT"
+  head "https://github.com/oven-sh/bun.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(/bun-v?(\d+(?:\.\d+)+)/i)
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "esbuild" => :build
+  depends_on "gnu-sed" => :build
+  depends_on "go" => :build
+  depends_on "libtool" => :build
+  depends_on "llvm@13" => :build
```

Comment:
This isn't build-only. It uses libc from here

Reply:
yup this was unintentional linkage, fixed